docs/pywbemclicmdshelp.rst

```diff
@@ -858,13 +861,12 @@ The following defines the help output for the `pywbemcli connection list --help`
 
       List the WBEM connection definitions.
 
-      List the WBEM connection definitions in the connections file as a table.
-
-      In the table, the current connection is marked with an "*" after its name.
-
-      Examples:
+      This command displays all entries in the connections file and the current
+      connection if it is not in the connections file as a table using the
+      command line output_format to define the table format.
```

Comment:
On "using the command line output_format to define the table format.":
1. Don't describe this using the name of an internal variable (output_format).
2. I don't think we need to repeat every time we say an output is a table that the format of the table can be controlled.

I suggest to drop the quoted part of the sentence.